docs/guidelines/Colors.md

```diff
@@ -7,46 +7,37 @@ title: Colors
 
 **Page Summary**
 
-* Table of contents
-{:toc}
```

Comment:
For which reason did you remove the automatic table of contents?

Same question for `Typography.md`.